src/Functions/toDaysSinceYearZero.cpp

```diff
@@ -58,38 +57,41 @@ class FunctionToDaysSinceYearZero : public IFunction
         WhichDataType which(from_type);
 
         if (which.isDate())
-            return DateTimeTransformImpl<DataTypeDate, ResultType, ToDaysSinceYearZeroImpl>::execute(arguments, result_type, input_rows_count);
+            return DateTimeTransformImpl<DataTypeDate, ResultType, ToDaysSinceYearZeroImpl>::execute(
+                arguments, result_type, input_rows_count);
         else if (which.isDate32())
-            return DateTimeTransformImpl<DataTypeDate32, ResultType, ToDaysSinceYearZeroImpl>::execute(arguments, result_type, input_rows_count);
+            return DateTimeTransformImpl<DataTypeDate32, ResultType, ToDaysSinceYearZeroImpl>::execute(
+                arguments, result_type, input_rows_count);
         else if (which.isDateTime())
-            return DateTimeTransformImpl<DataTypeDateTime, ResultType, ToDaysSinceYearZeroImpl>::execute(arguments, result_type, input_rows_count);
+            return DateTimeTransformImpl<DataTypeDateTime, ResultType, ToDaysSinceYearZeroImpl>::execute(
```

Comment:
Needs tests: tests/queries/0_stateless/02874_toDaysSinceYearZero.sql

Reply:
yep, it is there already: :)
https://github.com/ClickHouse/ClickHouse/blob/4d3b127a1ff370fd5cee3dccdc2456163ad816ed/tests/queries/0_stateless/02874_toDaysSinceYearZero.sql